### ISSUE TITLE
cypress-cli install has -d option that seems useless

### ISSUE THREAD

**Issue by jhump** [cypress-io/cypress#389]
There is a `-d` argument to `cypress install` to control where cypress gets installed. However, there does not seem to be any way to get subsequent commands (e.g. invocations of `cypress run` or `cypress open`) to actually use that install directory.

So, after installing to a custom location, if I run `cypress version` I get this error message (obviously I am running on OS X):

```
Unhandled rejection Error: ENOENT: no such file or directory, stat '/Applications/Cypress.app/Contents/MacOS/Cypress'
  at Error (native)
```
(The error actually prints 2x, but that is likely beside the point.)

Other commands, like `cypress open` show this one:

```
 -Error- 
The Cypress App could not be found.
Expected the app to be found here: /Applications/Cypress.app

To fix this do (one) of the following:
1. Reinstall Cypress with: cypress install
2. If Cypress is stored in another location, move it to the expected location
```

I was hoping that maybe it would work if the custom directory were `~/Applications`, but still no go.

I think there ideally would be an environment variable (e.g. `CYPRESS_HOME`) that all other commands would inspect to discover where cypress is installed. If there is such an environment variable, I cannot find any documentation about it. (Just in case I guessed it correctly, I did actually try setting `CYPRESS_HOME`, but that had no effect.)

In addition to adding support for such an environment variable, `cypress version` should be updated to show a better error message (more like the one shown by `cypress open`) and the nicer-looking message should offer a 3rd option: set the environment variable and re-run.

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
